frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from '~/libs/components/components.js';
+import { useAppSelector } from '~/libs/hooks/hooks.js';
+
+import avatar from '../../assets/img/avatar.jpg';
```

Comment:
```suggestion
import avatar from '~/assets/img/avatar.jpg';
```
I think, it would be more neat

Reply:
changed

Reply:
Not changed